test/run-status.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var path = require('path');
+var test = require('tap').test;
+var chalk = require('chalk');
+var figures = require('figures');
+var RunStatus = require('../lib/run-status');
+
+var sep = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
+
+test('requires new', function (t) {
+	var runStatus = RunStatus;
+	t.throws(function () {
+		runStatus({});
+	}, 'Class constructor RunStatus cannot be invoked without \'new\'');
+	t.end();
+});
+
+test('prefixTitle returns empty if prefixTitles == false', function (t) {
+	var runStatus = new RunStatus({prefixTitles: false});
+	t.is(runStatus.prefixTitle('test/run-status.js'), '');
+	t.end();
+});
+
+test('prefixTitle removes base if found at start of path', function (t) {
+	var runStatus = new RunStatus({base: 'test' + path.sep});
+	t.is(runStatus.prefixTitle('test/run-status.js'), 'run-status' + sep);
+	t.end();
+});
+
+test('prefixTitle does not remove base if found but not at start of path', function (t) {
+	var runStatus = new RunStatus({base: path.sep});
```

Comment:
@nfcampos yes makes sense. So then you could test with `bar/foo/baz.js` right?

`prefixTitle` replaces slashes so IMO it's not a great `base` value for this test.